### ISSUE TITLE
Size Classes

### ISSUE THREAD

**Issue by charliemcculloch** [matt-curtis/Fluid-for-Sketch#36]
Are there any plans to include iOS size classes? Or am I missing something. At the moment it seems like constraints are globally applied. To clarify: It would be really useful to have differing layouts depending on size class.

**Comment by matt-curtis** [comment]
Yes, constraints are 'globally' applied (as in there only exists the constraints set on a layer, rather than 'sets' of constraints). I've thought about this. There's no ETA or immediate plans to implement it, but it's under consideration.

**Comment by charliemcculloch** [comment]
I would love to see it in the plugin. At the very least being able to set the size of the Preview artboards would be good, because then you could preview how one size class's constraints behaves across specific devices within the class, eg one artboard for iPhones and another for iPads.

**Comment by imcatnoone** [comment]
Thanks a ton for the input, @charliemcculloch! I can totally see the use for that, and it's definitely something Matt and I are considering.

**Comment by lmerriam** [comment]
Came here to post the same suggestions. I could really use conditional size classes!
